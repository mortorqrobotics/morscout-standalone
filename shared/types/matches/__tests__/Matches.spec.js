import { loadAllMatches, updated } from "../index";

test("load All Matches stays the same", () => {
  expect(loadAllMatches).toMatchSnapshot();
});

test("updated Matches stays the same", () => {
  expect(updated).toMatchSnapshot();
});
