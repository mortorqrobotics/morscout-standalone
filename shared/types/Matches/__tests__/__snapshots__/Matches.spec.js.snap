// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`load All Matches stays the same 1`] = `"server/getMatches"`;

exports[`updated Matches stays the same 1`] = `"UPDATED_MATCHES"`;
